app/assets/javascripts/FormMessageController.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Created by elisahilprecht on 09/04/15.
+ */
+window.FormMessageController = {};
+(function(FormMessageController){
+
+  var getElementsByClassName = function(node, classname) {
+    var a = [];
+    var re = new RegExp('(^| )'+classname+'( |$)');
+    var els = node.getElementsByTagName("*");
+    for(var i=0,j=els.length; i<j; i++)
+      if(re.test(els[i].className))a.push(els[i]);
+    return a;
+  };
```

Comment:
It is because it does not work in the tests. We get an error there. Joé told me to replace it by something, which is not failing.